components/Recipe/index.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "styled-components";
+
+const RecipeContainer = styled.div`
+  flex-basis: calc(50% - 10px);
+  display: flex;
+  flex-direction: column;
+`;
+
+const RecipeImage = styled.img`
+  width: 100%;
+  height: 200px;
+  object-fit: cover;
+  border-radius: 15px;
+`;
+
+const RecipeTitle = styled.p`
+  font-weight: bold;
+  text-align: left;
+  margin: 5px 0;
+`;
+
+const RecipeSubtitle = styled.p`
+  font-style: italic;
+  text-align: left;
+  margin: 3px 0;
+`;
```

Comment:
here you have two styled components for the same tag, you could combine this two components to one and pass a props that decides on which styling shoud be used.

Reply:
I'll adjust it